pkgs/applications/networking/cluster/terraform-providers/default.nix

```diff
@@ -7,7 +8,23 @@
 let
   list = lib.importJSON ./providers.json;
 
-  toDrv = name: data:
+  buildWithGoModule = name: data:
```

Comment:
The `name` is not used. Am I missing something? :thinking:

Reply:
Excellent spot, fixed in both `buildWithGoModule` and `buildWithGoPackage` :bow:

Reply:
@kalbasit would you be able to give this a once over. I've address all outstanding comments :bow: